gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -1808,26 +1799,17 @@ def test_{{ method_name }}_rest_no_http_options():
 {% endmacro %}
 
 
-{% macro empty_call_test(method, method_name, service, api, uuid4_re, is_async=False) %}
+{% macro method_call_test(test_name, method, service, api, request, is_async=False) %}
```

Comment:
The comments within this macro suggest it's gRPC-only, so we should either clarify that in the name or add a TODO to say our intent is to expand this to REST as well.

Reply:
Good catch! This test only supports gRPC. I filed a bug to follow up on adding support for REST: https://github.com/googleapis/gapic-generator-python/issues/2159